### PR TITLE
fix(theme): form inputs should have no appearance

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Autoprefixer don't deal with appearance property prefixes

**What is the new behavior?**
Force prefix for appearance

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Other information**:
I didn't set any Firefox prefix for `radio` and `checkbox` elements since it does not support pseudo element.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://186.talend.surge.sh)

File: packages/theme/src/theme/_forms.scss [hunk 1]
```diff
@@ -118,6 +120,7 @@ form {
 		border: none;
 		background-color: transparent;
 		appearance: none;
+		-webkit-appearance: none;
```
Comment (charlesnguyen) [inline packages/theme/src/theme/_forms.scss]: Why don't you add "-moz-appearance: none;" here?

File: packages/theme/src/theme/_forms.scss [hunk 2]
```diff
@@ -173,6 +176,7 @@ form {
 		border: none;
 		margin-bottom: -4px;
 		appearance: none;
+		-webkit-appearance: none;
```
Comment (charlesnguyen) [inline packages/theme/src/theme/_forms.scss]: Why don't you add "-moz-appearance: none;" here?

**Comment by jsomsanith** [comment]
Why do you add those prefixes instead of using autoprefixer ?

File: packages/theme/src/theme/_forms.scss [hunk 3]
```diff
@@ -61,6 +61,8 @@ form {
 		border: none;
 		border-radius: 0;
 		appearance: none;
+		-moz-appearance: none;
+		-webkit-appearance: none;
```
Comment (talend-glorieux) [inline packages/theme/src/theme/_forms.scss]: Don't we need an extra `-ms-appearance: none;`?

**Comment by frassinier** [comment]
@charlesnguyen Look at the PR description
> I didn't set any Firefox prefix for radio and checkbox elements since it does not support pseudo element.
@jsomsanith Because Autoprefixer skips it, just a quick fix here
@talend-glorieux Microsoft has no support :(

**Comment by jsomsanith** [comment]
Problem solved with #193

**Pull request closed without merging**